packages/website/ts/components/wallet/wallet.tsx

```diff
@@ -458,21 +475,35 @@ export class Wallet extends React.Component<WalletProps, WalletState> {
             />
         );
     }
-    private _renderAmount(amount: BigNumber, decimals: number, symbol: string): React.ReactNode {
+    private _renderAmount(isLoaded: boolean, amount: BigNumber, decimals: number, symbol: string): React.ReactNode {
```

Comment:
nit: would be nicer to have these `isLoading` booleans as last optional arguments that default to false or something